src/main/java/seedu/address/storage/JsonStaffListStorage.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.storage;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.commons.util.FileUtil;
+import seedu.address.commons.util.JsonUtil;
+import seedu.address.model.ReadOnlyStaffList;
+
+/**
+ * A class to access StaffList data stored as a json file on the hard disk.
+ */
+public class JsonStaffListStorage implements StaffListStorage {
+
```

Comment:
Not sure if this class is necessary, because maybe we can just store the staffList in projects